lms/templates/dashboard.html

```diff
@@ -118,6 +118,14 @@
       </div>
     %endif
 
+    %if course_upgrade_message:
+    <div class="course-upgrade aa-icon info" role="alert">
+      <div class="message-copy" >
+        ${course_upgrade_message | n,  decode.utf8}
```

Comment:
Decoding of message should be handling inside view

Reply:
Requested changes have been done.